dvc/visualization.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, Iterable, List
```

Comment:
for the record: `visualization.py` sounds weird, discussed possibly renaming it and maybe even moving it somewhere else to utils or somtehing.

Reply:
moved to `dvc/utils/html.py`, also refactored embedding of plots and metrics.